config/serverless.es.yml

```diff
@@ -0,0 +1 @@
+uiSettings.overrides.defaultRoute: /app/enterprise_search/content/search_indices
```

Comment:
I added this setting to change the landing page. This would make it super obvious that we are loading different project type files.

Reply:
This is in the PoC, we'll be doing this for IC2, so this works.